src/renderer/i18n/locales/en-US.js

```diff
@@ -468,15 +468,18 @@ export default {
       STEP2: {
         INSTRUCTIONS: {
           HEADER: 'Network selection',
-          TEXT: 'Choose the network of this profile.'
+          TEXT: 'Choose the network of this profile.',
+          CUSTOM_NETWORK: 'or you can choose a custom network',
+          CUSTOM_NETWORK_EXPLAIN: 'You can select your custom network or choose any of the available networks.'
         },
         NAME: 'Profile name',
         TITLE: '2. Network'
       },
       STEP3: {
         INSTRUCTIONS: {
           HEADER: 'Appearance',
-          TEXT: 'Customize this application selecting one of our themes and backgrounds.'
+          TEXT: 'Customize this application selecting one of our themes and backgrounds.',
```

Comment:
not seeing this one either in the UI, also maybe add 'by' to the sentence: `... this application by selecting ...`